### PR TITLE
Fix incorrect content of device name info dialog

### PR DESCRIPTION
**Pull request by sabercodic**
This change is [Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5204)

### CODE REVIEW

**Review by Rawa — CHANGES_REQUESTED** [review]
> Reviewed 1 of 21 files at r2.
> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5204)* status: 0 of 28 files reviewed, 4 unresolved discussions (waiting on @sabercodic)

___
*[`android/app/src/main/kotlin/net/mullvad/mullvadvpn/compose/dialog/DeviceNameInfoDialog.kt` line 9 at r2](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5204#-NfPcusE5ZBjpiMLb2L7:-NfPcusE5ZBjpiMLb2L8:bp92agt) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/f59dea5db1cdbed7c2ec5f4ce188ddc337036ebd/android/app/src/main/kotlin/net/mullvad/mullvadvpn/compose/dialog/DeviceNameInfoDialog.kt#L9)):*
> ```Kotlin
> @Composable
> fun DeviceNameInfoDialog(onDismiss: () -> Unit) {
>     InfoDialog(
> ```

This dialog now exists here: 
https://github.com/mullvad/mullvadvpn-app/blob/6541297ffc1b1a4f2330b589cadfe81fffdd0e4a/android/app/src/main/kotlin/net/mullvad/mullvadvpn/compose/screen/WelcomeScreen.kt#L256
Let's migrate that one over.
___
*[`android/app/src/main/kotlin/net/mullvad/mullvadvpn/compose/screen/AccountScreen.kt` line 61 at r2](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5204#-NfPe4U2-XWJKgBvNuEb:-NfPe4U2-XWJKgBvNuEc:b-jljncv) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/f59dea5db1cdbed7c2ec5f4ce188ddc337036ebd/android/app/src/main/kotlin/net/mullvad/mullvadvpn/compose/screen/AccountScreen.kt#L61)):*
> ```Kotlin
>                 accountNumber = "1234123412341234",
>                 accountExpiry = null,
>                 dialogState = AccountScreenDialogState.NoDialog
> ```

Generally I like keeping all the state in the ViewState. But in this case I think we can avoid it. When we migrate to compose navigation, a dialog with just info would be a destination (much like a dialog fragment) thus it would not have to be in the ViewState.

For now I think we can simplify the showing and non showing of the dialog just in compose.
```var showDeviceNameInfoDialog by remember { mutableStateOf(false) }```
and then just set to false on dismiss:
https://github.com/mullvad/mullvadvpn-app/blob/6541297ffc1b1a4f2330b589cadfe81fffdd0e4a/android/app/src/main/kotlin/net/mullvad/mullvadvpn/compose/screen/WelcomeScreen.kt#L265C20-L265C20

Later once we have the navigation in compose we can replace enabling of `showDeviceNameInfoDialog` ( https://github.com/mullvad/mullvadvpn-app/blob/6541297ffc1b1a4f2330b589cadfe81fffdd0e4a/android/app/src/main/kotlin/net/mullvad/mullvadvpn/compose/screen/WelcomeScreen.kt#L247C16-L247C16 ) with just a call to navigate to the dialog.
___
*[`android/app/src/main/kotlin/net/mullvad/mullvadvpn/compose/screen/AccountScreen.kt` line 155 at r2](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5204#-NfPfUkn750J9LF-wNoc:-NfPfUkn750J9LF-wNod:b-xzm9wu) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/f59dea5db1cdbed7c2ec5f4ce188ddc337036ebd/android/app/src/main/kotlin/net/mullvad/mullvadvpn/compose/screen/AccountScreen.kt#L155)):*
> ```Kotlin
>                         contentDescription = null,
>                         tint = MaterialTheme.colorScheme.inverseSurface
>                     )
> ```

We should use a IconButton, if it is a clickable button. https://github.com/mullvad/mullvadvpn-app/blob/6541297ffc1b1a4f2330b589cadfe81fffdd0e4a/android/app/src/main/kotlin/net/mullvad/mullvadvpn/compose/screen/WelcomeScreen.kt#L245 It will help us with touch target size etc.
___
*[`android/lib/resource/src/main/res/values/strings.xml` line 224 at r1](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5204#-NfPgHqQD-tt9qNKu5QB:-NfPgHqQD-tt9qNKu5QC:b97e1uq) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/13ef02c1839706da1201e9e5fdcc9e7984e3eee1/android/lib/resource/src/main/res/values/strings.xml#L224)):*
> ```XML
>     <string name="device_name_info_part1">This is the name assigned to the device. Each device logged in on a Mullvad account gets a unique name that helps you identify it when you manage your devices in the app or on the website.</string>
>     <string name="device_name_info_part2">You can have up to 5 devices logged in on one Mullvad account.</string>
>     <string name="device_name_info_part3">If you log out, the device and the device name is removed. When you log back in again, the device will get a new name.</string>
> ```

These are already in there now with `device_name_info_first_paragraph` etc. lets remove them.

**Review by sabercodic — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5204)* status: 0 of 28 files reviewed, 4 unresolved discussions (waiting on @Rawa)

___
*[`android/app/src/main/kotlin/net/mullvad/mullvadvpn/compose/dialog/DeviceNameInfoDialog.kt` line 9 at r2](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5204#-NfPcusE5ZBjpiMLb2L7:-NfPiw94DMCmTq6jBgnN:b-896fix) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/f59dea5db1cdbed7c2ec5f4ce188ddc337036ebd/android/app/src/main/kotlin/net/mullvad/mullvadvpn/compose/dialog/DeviceNameInfoDialog.kt#L9)):*
<details><summary><i>Previously, Rawa (David Göransson) wrote…</i></summary><blockquote>

This dialog now exists here: 
https://github.com/mullvad/mullvadvpn-app/blob/6541297ffc1b1a4f2330b589cadfe81fffdd0e4a/android/app/src/main/kotlin/net/mullvad/mullvadvpn/compose/screen/WelcomeScreen.kt#L256
Let's migrate that one over.
</blockquote></details>

Done.
___
*[`android/app/src/main/kotlin/net/mullvad/mullvadvpn/compose/screen/AccountScreen.kt` line 61 at r2](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5204#-NfPe4U2-XWJKgBvNuEb:-NfPnF3q9ec9lqY-hnJr:b-896fix) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/f59dea5db1cdbed7c2ec5f4ce188ddc337036ebd/android/app/src/main/kotlin/net/mullvad/mullvadvpn/compose/screen/AccountScreen.kt#L61)):*
<details><summary><i>Previously, Rawa (David Göransson) wrote…</i></summary><blockquote>

Generally I like keeping all the state in the ViewState. But in this case I think we can avoid it. When we migrate to compose navigation, a dialog with just info would be a destination (much like a dialog fragment) thus it would not have to be in the ViewState.

For now I think we can simplify the showing and non showing of the dialog just in compose.
```var showDeviceNameInfoDialog by remember { mutableStateOf(false) }```
and then just set to false on dismiss:
https://github.com/mullvad/mullvadvpn-app/blob/6541297ffc1b1a4f2330b589cadfe81fffdd0e4a/android/app/src/main/kotlin/net/mullvad/mullvadvpn/compose/screen/WelcomeScreen.kt#L265C20-L265C20

Later once we have the navigation in compose we can replace enabling of `showDeviceNameInfoDialog` ( https://github.com/mullvad/mullvadvpn-app/blob/6541297ffc1b1a4f2330b589cadfe81fffdd0e4a/android/app/src/main/kotlin/net/mullvad/mullvadvpn/compose/screen/WelcomeScreen.kt#L247C16-L247C16 ) with just a call to navigate to the dialog.
</blockquote></details>

Done.
___
*[`android/app/src/main/kotlin/net/mullvad/mullvadvpn/compose/screen/AccountScreen.kt` line 155 at r2](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5204#-NfPfUkn750J9LF-wNoc:-NfPlQI74o4_N_yn-IVJ:b-896fix) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/f59dea5db1cdbed7c2ec5f4ce188ddc337036ebd/android/app/src/main/kotlin/net/mullvad/mullvadvpn/compose/screen/AccountScreen.kt#L155)):*
<details><summary><i>Previously, Rawa (David Göransson) wrote…</i></summary><blockquote>

We should use a IconButton, if it is a clickable button. https://github.com/mullvad/mullvadvpn-app/blob/6541297ffc1b1a4f2330b589cadfe81fffdd0e4a/android/app/src/main/kotlin/net/mullvad/mullvadvpn/compose/screen/WelcomeScreen.kt#L245 It will help us with touch target size etc.
</blockquote></details>

Done.
___
*[`android/lib/resource/src/main/res/values/strings.xml` line 224 at r1](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5204#-NfPgHqQD-tt9qNKu5QB:-NfPlUq08Hq2MrwWsGaq:b-896fix) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/13ef02c1839706da1201e9e5fdcc9e7984e3eee1/android/lib/resource/src/main/res/values/strings.xml#L224)):*
<details><summary><i>Previously, Rawa (David Göransson) wrote…</i></summary><blockquote>

These are already in there now with `device_name_info_first_paragraph` etc. lets remove them.
</blockquote></details>

Done.

**Review by Pururun — CHANGES_REQUESTED** [review]
> Reviewed 3 of 28 files at r1, 20 of 21 files at r2, 1 of 1 files at r3, 5 of 5 files at r4, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5204)* status: all files reviewed, 5 unresolved discussions (waiting on @Rawa and @sabercodic)

___
*[`android/app/src/main/kotlin/net/mullvad/mullvadvpn/viewmodel/AccountViewModel.kt` line 39 at r4](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5204#-NfQ3PTr51ycDdcOMeiU:-NfQ3PTr51ycDdcOMeiV:bajugft) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/99db92e0fcd975101cb501c641889b0f61fffd41/android/app/src/main/kotlin/net/mullvad/mullvadvpn/viewmodel/AccountViewModel.kt#L39)):*
> ```Kotlin
>             .stateIn(viewModelScope, SharingStarted.WhileSubscribed(), AccountUiState.default())
>     val uiState =
>         vmState.stateIn(viewModelScope, SharingStarted.WhileSubscribed(), AccountUiState.default())
> ```

We could probably remove `vmState`, it serves no purpose.

**Review by Rawa — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5204)* status: all files reviewed, 5 unresolved discussions (waiting on @sabercodic)

___
*[`android/app/src/main/kotlin/net/mullvad/mullvadvpn/ui/fragment/AccountFragment.kt` line 40 at r4](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5204#-NfQ3NOlEqZOnOyupdgt:-NfQ3NOlEqZOnOyupdgu:bcv502w) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/99db92e0fcd975101cb501c641889b0f61fffd41/android/app/src/main/kotlin/net/mullvad/mullvadvpn/ui/fragment/AccountFragment.kt#L40)):*
> ```Kotlin
>                         onManageAccountClick = vm::onManageAccountClick,
>                         onLogoutClick = vm::onLogoutClick,
>                         onBackClick = { activity?.onBackPressedDispatcher?.onBackPressed() }
> ```

Nice :+1: 
___
*[`android/app/src/main/kotlin/net/mullvad/mullvadvpn/viewmodel/AccountUiState.kt` line 7 at r4](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5204#-NfQ4EQt-s-lJ0yY4wdj:-NfQ4EQt-s-lJ0yY4wdk:b-8gyaod) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/99db92e0fcd975101cb501c641889b0f61fffd41/android/app/src/main/kotlin/net/mullvad/mullvadvpn/viewmodel/AccountUiState.kt#L7)):*
> ```Kotlin
> import org.joda.time.DateTime
> 
> data class AccountUiState(
> ```

Why did we move this class to `viewmodel` package?
___
*[`android/app/src/main/kotlin/net/mullvad/mullvadvpn/viewmodel/AccountUiState.kt` line 9 at r4](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5204#-NfQ4Z0-0ME6xR3Q-Xka:-NfQ4Z0-0ME6xR3Q-Xkb:bc8bf0l) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/99db92e0fcd975101cb501c641889b0f61fffd41/android/app/src/main/kotlin/net/mullvad/mullvadvpn/viewmodel/AccountUiState.kt#L9)):*
> ```Kotlin
> data class AccountUiState(
>     val deviceName: String?,
>     val accountNumber: String?,
> ```

I see that deviceName & accountNumber wen't from non-null to nullable, is that avoidable? If we don't have an account number or a device name that it maybe should be fatal for the AccountScreen?

**Review by sabercodic — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5204)* status: all files reviewed, 4 unresolved discussions (waiting on @Pururun and @Rawa)

___
*[`android/app/src/main/kotlin/net/mullvad/mullvadvpn/viewmodel/AccountUiState.kt` line 7 at r4](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5204#-NfQ4EQt-s-lJ0yY4wdj:-NfQAdCa04mAjTQkTVSb:b-896fix) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/99db92e0fcd975101cb501c641889b0f61fffd41/android/app/src/main/kotlin/net/mullvad/mullvadvpn/viewmodel/AccountUiState.kt#L7)):*
<details><summary><i>Previously, Rawa (David Göransson) wrote…</i></summary><blockquote>

Why did we move this class to `viewmodel` package?
</blockquote></details>

Done.
___
*[`android/app/src/main/kotlin/net/mullvad/mullvadvpn/viewmodel/AccountUiState.kt` line 9 at r4](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5204#-NfQ4Z0-0ME6xR3Q-Xka:-NfQB1wY5QRZbpyhG2AU:besbews) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/99db92e0fcd975101cb501c641889b0f61fffd41/android/app/src/main/kotlin/net/mullvad/mullvadvpn/viewmodel/AccountUiState.kt#L9)):*
<details><summary><i>Previously, Rawa (David Göransson) wrote…</i></summary><blockquote>

I see that deviceName & accountNumber wen't from non-null to nullable, is that avoidable? If we don't have an account number or a device name that it maybe should be fatal for the AccountScreen?
</blockquote></details>

there was a feedback from Albin on previous PR and this was the result
___
*[`android/app/src/main/kotlin/net/mullvad/mullvadvpn/viewmodel/AccountViewModel.kt` line 39 at r4](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5204#-NfQ3PTr51ycDdcOMeiU:-NfQA7VPAqBlyb4Sm5Mz:bsoo5jr) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/99db92e0fcd975101cb501c641889b0f61fffd41/android/app/src/main/kotlin/net/mullvad/mullvadvpn/viewmodel/AccountViewModel.kt#L39)):*
<details><summary><i>Previously, Pururun (Jonatan Rhodin) wrote…</i></summary><blockquote>

We could probably remove `vmState`, it serves no purpose.
</blockquote></details>

should I do it in this PR?

**Review by Pururun — CHANGES_REQUESTED** [review]
> Reviewed 2 of 2 files at r5, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5204)* status: all files reviewed, 4 unresolved discussions (waiting on @Rawa and @sabercodic)

___
*[`android/app/src/main/kotlin/net/mullvad/mullvadvpn/viewmodel/AccountViewModel.kt` line 39 at r4](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5204#-NfQ3PTr51ycDdcOMeiU:-NfQE5nGA9KOj360EkRm:b-bunh29) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/99db92e0fcd975101cb501c641889b0f61fffd41/android/app/src/main/kotlin/net/mullvad/mullvadvpn/viewmodel/AccountViewModel.kt#L39)):*
<details><summary><i>Previously, sabercodic wrote…</i></summary><blockquote>

should I do it in this PR?
</blockquote></details>

I would say so yes

**Review by albin-mullvad — COMMENTED** [review]
> Reviewed 1 of 2 files at r5.
> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5204)* status: all files reviewed, 4 unresolved discussions (waiting on @Rawa and @sabercodic)

___
*[`android/app/src/main/kotlin/net/mullvad/mullvadvpn/viewmodel/AccountUiState.kt` line 9 at r4](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5204#-NfQ4Z0-0ME6xR3Q-Xka:-NfQFt1yD-UYN-C4q_BP:b-rx1w69) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/99db92e0fcd975101cb501c641889b0f61fffd41/android/app/src/main/kotlin/net/mullvad/mullvadvpn/viewmodel/AccountUiState.kt#L9)):*
<details><summary><i>Previously, sabercodic wrote…</i></summary><blockquote>

there was a feedback from Albin on previous PR and this was the result
</blockquote></details>

@sabercodic my feedback was that they should be unified since there was a mix of nullable and non-nullable similar properties.

@Rawa it's not a fatal state since we show spinners if the data is not present.

So to summarize, we should use a common unified way of representing the lack of data across these properties (which should result in a spinner), which can either be by having them be nullable or using some other mechanism.

**Review by sabercodic — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5204)* status: all files reviewed, 4 unresolved discussions (waiting on @Pururun and @Rawa)

___
*[`android/app/src/main/kotlin/net/mullvad/mullvadvpn/viewmodel/AccountViewModel.kt` line 39 at r4](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5204#-NfQ3PTr51ycDdcOMeiU:-NfQcJkU40RzBO1iauFq:b-896fix) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/99db92e0fcd975101cb501c641889b0f61fffd41/android/app/src/main/kotlin/net/mullvad/mullvadvpn/viewmodel/AccountViewModel.kt#L39)):*
<details><summary><i>Previously, Pururun (Jonatan Rhodin) wrote…</i></summary><blockquote>

I would say so yes
</blockquote></details>

Done.
___
*[`android/app/src/main/kotlin/net/mullvad/mullvadvpn/viewmodel/AccountUiState.kt` line 9 at r4](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5204#-NfQ4Z0-0ME6xR3Q-Xka:-NfQcLFO0ITqiZSRiXr4:bh6bwc6) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/99db92e0fcd975101cb501c641889b0f61fffd41/android/app/src/main/kotlin/net/mullvad/mullvadvpn/viewmodel/AccountUiState.kt#L9)):*
<details><summary><i>Previously, albin-mullvad wrote…</i></summary><blockquote>

@sabercodic my feedback was that they should be unified since there was a mix of nullable and non-nullable similar properties.

@Rawa it's not a fatal state since we show spinners if the data is not present.

So to summarize, we should use a common unified way of representing the lack of data across these properties (which should result in a spinner), which can either be by having them be nullable or using some other mechanism.
</blockquote></details>

Spinner will shows in case of null (before loading the actual data)

**Review by Rawa — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5204)* status: all files reviewed, 3 unresolved discussions (waiting on @albin-mullvad, @Pururun, and @sabercodic)

___
*[`android/app/src/main/kotlin/net/mullvad/mullvadvpn/viewmodel/AccountUiState.kt` line 9 at r4](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5204#-NfQ4Z0-0ME6xR3Q-Xka:-NfQlAniCfcsgMcYvo5S:b-8fx2u7) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/99db92e0fcd975101cb501c641889b0f61fffd41/android/app/src/main/kotlin/net/mullvad/mullvadvpn/viewmodel/AccountUiState.kt#L9)):*
<details><summary><i>Previously, sabercodic wrote…</i></summary><blockquote>

Spinner will shows in case of null (before loading the actual data)
</blockquote></details>

:+1:

**Review by Rawa — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5204)* status: all files reviewed, 4 unresolved discussions (waiting on @albin-mullvad, @Pururun, and @sabercodic)

___
*[`android/app/src/main/kotlin/net/mullvad/mullvadvpn/compose/dialog/DeviceNameInfoDialog.kt` line 16 at r5](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5204#-NfQmhmVCMRKNOj-1oOA:-NfQmhmVCMRKNOj-1oOB:biliunk) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/9491348efd5abed95f9bf31a20e9c40228455b99/android/app/src/main/kotlin/net/mullvad/mullvadvpn/compose/dialog/DeviceNameInfoDialog.kt#L16)):*
> ```Kotlin
>                 appendLine(stringResource(id = R.string.device_name_info_second_paragraph))
>                 appendLine()
>                 appendLine(stringResource(id = R.string.device_name_info_third_paragraph))
> ```

Last thing to fix! This is my fault, the last one should be `append` not `appendLine`

**Review by sabercodic — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5204)* status: 27 of 29 files reviewed, 2 unresolved discussions (waiting on @Pururun and @Rawa)

___
*[`android/app/src/main/kotlin/net/mullvad/mullvadvpn/compose/dialog/DeviceNameInfoDialog.kt` line 16 at r5](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5204#-NfQmhmVCMRKNOj-1oOA:-NfQowyBAdyZc28vzeu0:b-896fix) ([raw file](https://github.com/mullvad/mullvadvpn-app/blob/9491348efd5abed95f9bf31a20e9c40228455b99/android/app/src/main/kotlin/net/mullvad/mullvadvpn/compose/dialog/DeviceNameInfoDialog.kt#L16)):*
<details><summary><i>Previously, Rawa (David Göransson) wrote…</i></summary><blockquote>

Last thing to fix! This is my fault, the last one should be `append` not `appendLine`
</blockquote></details>

Done.

**Review by Rawa — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5204#-:-NfQvJQe4gza7AW69Yv1:bnfp4nl)* status: 27 of 29 files reviewed, 1 unresolved discussion (waiting on @Pururun)

**Review by Rawa — COMMENTED** [review]
Good job! :+1: 

> *[Reviewable](https://reviewable.io/reviews/mullvad/mullvadvpn-app/5204#-:-NfQvKvB6I-OLvJh9hpM:b-yrlqoo)* status: 27 of 29 files reviewed, 1 unresolved discussion (waiting on @Pururun)
